includes/class-activate.php

```
@@ -36,7 +36,24 @@ public static function init(): void {
+				$notices[] = 'foreign_dropin';
+			} else {
+				Advanced_Cache_Handler::create();
+			}
+
+			$wp_cache_notice = self::add_wp_cache_constant();
+			if ( is_string( $wp_cache_notice ) ) {
+				$notices[] = $wp_cache_notice;
+			}
+
+			if ( ! empty( $notices ) ) {
+				set_transient( 'wppo_activation_notices', array_unique( $notices ), WEEK_IN_SECONDS );
```

PHPStan (2.1.46)

[error] Constant WEEK_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)